scripts/run_benches_for_runtime.sh

```diff
@@ -44,32 +47,42 @@ pallets=(
 	pallet-asset-switch
 )
 
-
-
 # Add Peregrine-only pallets here!
 if [ "$runtime" = "peregrine" ]; then
   pallets+=(
 	pallet-sudo
   )
 fi
 
-echo "[+] Running all runtime benchmarks for $runtime --chain=$chain"
+echo "[+] Running all runtime benchmarks for \"$runtime\", \"--chain=$chain\" and profile \"$profile\""
 
 cargo build $standard_args
 
+if [ $profile == "dev" ]; then
+    target_folder="debug"
+	# We care about benchmark correctness, not accuracy.
+	additional_args="--steps=2 --repeat=1 --default-pov-mode=ignored --no-verify"
+else
+    target_folder=$profile
+	additional_args="--header=\"HEADER-GPL\" --template=\".maintain/runtime-weight-template.hbs\" --output=\"./runtimes/${runtime}/src/weights/\""
+fi
+
 for pallet in "${pallets[@]}"; do
 	echo "Runtime: $runtime. Pallet: $pallet"
 	# shellcheck disable=SC2086
-	./target/release/kilt-parachain benchmark pallet \
-		--template=".maintain/runtime-weight-template.hbs" \
-		--header="HEADER-GPL" \
+	./target/$target_folder/kilt-parachain benchmark pallet \
 		--wasm-execution=compiled \
```

Comment:
I think we can remove this flag

Reply:
Cleaned both up since `--wasm-execution=compiled` is the default and the `--execution=wasm` deprecated. I adjusted also the pallet benchmarking command to do the same: https://github.com/KILTprotocol/kilt-node/pull/731/commits/d8a43d450837b99ae27072ecfc8b2662c1d36042.